_data/master/navbars/getting-started.yml

```diff
@@ -33,6 +33,8 @@ toc:
       path: /getting-started/kubernetes/installation/vagrant/
   - title: Tutorials
     section:
+    - title: Using calicoctl
```

Comment:
I don't think this is the right place for a Using calicoctl guide - it should go in Usage.